contracts/multisig/src/error.rs

```diff
@@ -48,4 +48,7 @@ pub enum ContractError {
 
     #[error("key type mismatch")]
     KeyTypeMismatch,
+
+    #[error("caller is not authorized")]
+    Unauthorized { contract_address: Addr },
```

Comment:
I don't think we need the contract address here, since that address is the caller

Reply:
I found it helpful to see what address was trying to call the function when errors were thrown. Do you mean to rename `contract_address` to `caller`?